test/applySpec.js

```diff
@@ -21,6 +21,10 @@ describe('applySpec', function() {
        { unnested: 0, nested: { sum: 3 } });
   });
 
+  it('works with a spec defining a map key', function() {
+    eq(R.applySpec({map: R.always(1)})({}), {map: 1});
```

Comment:
I'd like it if this test actually used the value being passed in so we can verify the behavior is as expected.

Something like `R.applySpec({map: R.prop('a')})({a: 1})`